admin-front/src/components/Users.tsx

```diff
@@ -26,7 +26,7 @@ import axios from 'axios';
 const rolesEnum = {
   1: 'superadmin',
   2: 'editor',
-  3: 'creator',
+  3: 'moderator',
```

Comment:
This information shouldn't be hardcoded. Should be received from the backend (like how you receive the categories from BE). There's an endpoint for this already present: `/role`